config/default.yaml

```diff
@@ -143,6 +143,12 @@ bookends:
     teardown:
         - screwdriver-artifact-bookend
 
+notification:
```

Comment:
Kinda like how this is set to an empty object: https://github.com/screwdriver-cd/screwdriver/blob/2af654612965eeaac5719b0cff6c3f1c5393d3a8/config/default.yaml#L104